modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/PageRenderer/PageRenderer.soy

```diff
@@ -206,7 +206,7 @@
 						'fieldEdited': $_handleFieldEdited,
 						'fieldFocused': $_handleFieldFocused
 						] /}
-						{param ref: 'pageRendererFieldRef' + $field.instanceId + $rowIndex + $columnIndex + $pageIndex /}
+						{param ref: 'pageRendererFieldRef' + $field.instanceId + $field.name + $rowIndex + $columnIndex + $pageIndex /}
```

Comment:
@marcelabc, could you see the possibility of js template literals instead concatenation ?